OpenRA.Game/Traits/World/VisualizationMode.cs

```diff
@@ -0,0 +1,24 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA.Traits
+{
+	[Desc("Enables visualization commands. Attach this to the world actor.")]
+	public class VisualizationModeInfo : TraitInfo<VisualizationMode> { }
+
+	public class VisualizationMode
```

Comment:
Could we call these `DebugVisualizations(Info)`, and then `CombatGeometry`/`RenderGeometry`/`DepthBuffer`/`ActorTags`?  IMO these are a lot clearer and make the most of the context of now having our own dedicated container trait.